### PR TITLE
Follow style guide in a few test files

### PR DESCRIPTION
**Pull request by IndrajeetPatil**

### CODE REVIEW

**Comment by MichaelChirico** [comment]
would you mind splitting this up? it seems like two PRs in one

**Comment by IndrajeetPatil** [comment]
Done.

The current PR is only about styling now.

The additional test is in a new PR: https://github.com/r-lib/lintr/pull/1599

File: tests/testthat/test-closed_curly_linter.R [hunk 1]
```diff
@@ -3,60 +3,83 @@ test_that("returns the correct linting", {
...
   expect_warning(
     linter <- closed_curly_linter(),
     "Linter closed_curly_linter was deprecated",
     fixed = TRUE
   )
 
-  expect_lint("blah",
-              NULL,
-              linter)
+  expect_lint(
```
Comment (MichaelChirico) [inline tests/testthat/test-closed_curly_linter.R:15]: just inline this

File: tests/testthat/test-closed_curly_linter.R [hunk 2]
```diff
@@ -3,60 +3,83 @@ test_that("returns the correct linting", {
...
+    "blah",
+    NULL,
+    linter
+  )
 
-  expect_lint("a <- function() {\n}",
-              NULL,
-              linter)
+  expect_lint(
```
Comment (MichaelChirico) [inline tests/testthat/test-closed_curly_linter.R:21]: ditto, inlining should be fine

File: tests/testthat/test-function_return_linter.R [hunk 3]
```diff
@@ -65,14 +65,14 @@ test_that("function_return_linter blocks simple disallowed usages", {
   )
 
   side_effect_lines <-
-  expect_lint(
-    trim_some("
+    expect_lint(
```
Comment (MichaelChirico) [inline tests/testthat/test-function_return_linter.R:68]: move expect_lint() to the assignment line instead

**Review by MichaelChirico — APPROVED** [review]

**Pull request merged**